### ISSUE TITLE
Test output file name too long

### ISSUE THREAD

**Issue by masterleinad** [dealii/dealii#7660]
As reported in https://github.com/dealii/dealii/pull/7655#issuecomment-459136842 the length of the file name `tests/fe/fe_enriched_color_07.with_trilinos=true.with_muparser=true.with_trilinos_with_zoltan=true.with_petsc=true.with_petsc_with_complex=false.with_gsl=true.with_mpi=true.mpirun=2.output` causes problems when cloning for some users.

**Comment by drwells** [comment]
`with_trilinos` and `with_petsc` are both redundant since we have `with_trilinos_with_zoltan` and `with_petsc_with_complex`, right?

I think we can use `on` and `off` instead of `true` and `false`, which would also help.

**Comment by masterleinad** [comment]
@maieneuro What platform are you running on? Do you have an idea what the maximum allowed filename length is?

**Comment by drwells** [comment]
On Windows the maximum absolute file name length is 260 characters: this file name is 179 (!) characters.

**Comment by maieneuro** [comment]
I'm running on Ubuntu 18.04.
```
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.1 LTS
Release:	18.04
Codename:	bionic
```
The file system of the partition where I'm cloning the code is `ecryptfs`, which is possibly why the maximum file length is less than 255 bytes..
```
df -T .
Filesystem           Type     Mounted on
/home/maien/.Private ecryptfs /home/maien
```

**Comment by Rombur** [comment]
Yeah `ecryptfs` is the problem see [here](https://unix.stackexchange.com/questions/32795/what-is-the-maximum-allowed-filename-and-folder-size-with-ecryptfs). The maximum size is 140 characters.
